app/javascript/src/component_flow_connector_path.js

```diff
@@ -571,11 +571,12 @@ class ForwardDownBackwardUpPath extends FlowConnectorPath {
 
   constructor(points, config) {
     super(points, config);
+
     var dimensions = {
       forward1: Math.round(this.points.via_x - CURVE_SPACING),
       down: Math.round(utilities.difference(this.points.from_y, config.bottom) - (CURVE_SPACING * 2)),
       backward: Math.round(utilities.difference(this.points.from_x + this.points.via_x, this.points.to_x)),
-      up: Math.round(utilities.difference(config.bottom, this.points.from_y) + utilities.difference(this.points.from_y, this.points.to_y) - (CURVE_SPACING * 2)),
+      up: Math.round( utilities.difference(config.bottom, this.points.to_y) - (CURVE_SPACING * 2)),
```

Comment:
Because we always go down to the bottom in order to go back up, we can simply calculate the up path as the difference between the bottom and the 'to' item. i.e. simply the opposite of a down arrow (compare line 577 and 579)